### PR TITLE
EIP1559

### PR DESCRIPTION
**Pull request by richard-ramos**
Depends on https://github.com/status-im/status-lib/pull/12

### CODE REVIEW

**Comment by richard-ramos** [comment]
@carlfairclough mentioned in discord in regards to calculating priority ranges:
> On the implementers call it was suggested to grab the 10th %ile tip as the optimal suggestion, rather than the average. Which I’m inclined to agree with.  n gwei is the padding of the max fee to allow for changes in the base fee (i.e. if it continues to escalate at 12.5%, because of full blocks, then we need to get that transaction mined when auction conditions end.

Also, some values which I'm using based on https://github.com/status-im/status-react/pull/12221/files#diff-dc08595461416d2f109115beed393596521cd5e84a347f839bef8e8cc23c791eR65-R69 might need to be updated. 


> @carlfairclough: Definitely should just be for testing (the current values) IMO but others may disagree. Some people have said that 1 gwei won’t be mined but 1.2-1.5 will, for example. Also some have said that we should hardcode tips, but this doesn’t catch auction conditions. This is what we should be grabbing the %ile tips for

**Comment by anastasiyaig** [comment]
@richard-ramos could u please rebase? So txn history and collectibles are show up on mainnet? 

Also, i noticed that there is an error signing txn

Steps:

1. switch to ropsten
2. open simpledapp.eth dapp
3. click Assets
4. click request STT
5. click Sign with password


Screenshot 2021-08-05 at 14 57 09


```DBG 2021-08-05 14:55:08+03:00 callPrivateRPC                             topics="rpc" tid=9636906 file=core.nim:20 rpc_method=eth_estimateGas
CALLLING REMOTE eth_estimateGas
qrc:/app/AppLayouts/Chat/ChatColumn/ChatComponents/SignTransactionModal.qml:299: ReferenceError: group2 is not defined
DBG 2021-08-05 14:55:10+03:00 Verifying mailserver connection state...   topics="mailserver model" tid=9636956 file=model.nim:196
qrc:/app/AppLayouts/Chat/ChatColumn/ChatComponents/SignTransactionModal.qml:299: ReferenceError: group2 is not defined
DBG 2021-08-05 14:55:20+03:00 Verifying mailserver connection state...   topics="mailserver model" tid=9636956 file=model.nim:196

File: ui/app/AppLayouts/Browser/BrowserLayout.qml [hunk 1]
```diff
@@ -512,7 +514,7 @@ property Component sendTransactionModalComponent: SignTransactionModal {}
                 anchors.top: parent.top
                 anchors.topMargin: browserHeader.height
                 focus: true
-                url: "https://dap.ps"
+                url: "about:blank"
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Browser/BrowserLayout.qml:517]:
```suggestion
                url: "https://dap.ps"
```

**Comment by anastasiyaig** [comment]
@richard-ramos seems that beta 11 is not supporting the estimations 

The screen to the left is beta 11, screen to the right is current EIP1559 branch

Screenshot 2021-08-05 at 19 04 16

**Comment by anastasiyaig** [comment]
@richard-ramos can we please fix the representation of amounts because they are exceeding the borders? 

Also, even u showed me the calculation rules i dont get why the Low / Optimal / High suggestions are the same in EIP1559, while in Beta11 they are clearly different:

Screenshot 2021-08-05 at 21 56 55

**Comment by richard-ramos** [comment]
Fixed:
* [x] the representation of suggestion amounts are exceeding the borders
* [x] the Preview button should be renamed to Continue based on the figma
* [x] suggestion labels should not be in bold
* [x] remove white border at the top of this screen
* [x]  price limit should not be in red in case it is above minimum
* [x]  clicking Continue anyway does not close the modal
* [x]  gas validation is missing

**Comment by anastasiyaig** [comment]
The above mention UI issues are fixed. the decision of how to represent the gas estimations is not clear still

**Comment by anastasiyaig** [comment]
would be great if we can rebase this one too please @richard-ramos

**Comment by anastasiyaig** [comment]
@richard-ramos @iurimatias before merging that i want to clarify the thing that on mobile they did hide the slider and suggestions for the fee until better times (basically until the calculations for this will be more clear). Should we do the same on desktop before we merge this PR?

**Comment by richard-ramos** [comment]
Do you know what value is used by default for the fees in mobile?

**Comment by anastasiyaig** [comment]
@richard-ramos not sure (

**Comment by anastasiyaig** [comment]
lets merge it

**Pull request merged**